### PR TITLE
Uxedits

### PR DESCRIPTION
**Pull request by angeltapes**
I've made some style amendments to the index page and a comments section with a couple of suggestions. Let me know what you think. Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,24 +19,68 @@
...
+  <nav class="navbar navbar-inverse navbar-fixed-top">
+    <div class="container">
+        <div class="col-sm-6">
+            <img src="images/logo.jpg" id="se-logo">
+        </div>
+        <div class="col-sm-6">
+            <a class="navbar-brand" href="#">EUMIS Submissions</a>
+        </div>
+      </div>
+      <!--<div id="navbar" class="collapse navbar-collapse">
```
Comment (scotent) [inline index.html:31]: Don't commit commented out content. If it's needed it's needed.  If it's not. It's not.

File: index.html [hunk 2]
```diff
@@ -18,24 +19,68 @@
...
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>--><!--/.nav-collapse -->
+    </div>
+
+  </nav>
+
+<!--<div id="background-strip"></div>-->
+<div class="container" style="padding-top:2em;">
```
Comment (scotent) [inline index.html:43]: don't add inline styles.  Use classes

File: index.html [hunk 3]
```diff
@@ -18,24 +19,68 @@
...
+
 <div class="row">
         <div class="col-sm-6">
-            <h2>Submissions</h2>
+            <h2>EUMIS Submission management</h2>
+            <h2><small>Submit financial data about your projects online</small></h2>
```
Comment (scotent) [inline index.html:48]: h2 and small... sounds like an h3?

**Review by ghost — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (angeltapes) [reply index.html:31]: Removed, thanks

File: index.html (hunk 2 above)
Comment (angeltapes) [reply index.html:43]: Removed, thanks

File: index.html (hunk 3 above)
Comment (angeltapes) [reply index.html:48]: Thanks, I replaced this with 

`<h2>EUMIS Submission management</br><small>Submit financial data about your projects online</small></h2>`

**Comment by angeltapes** [comment]
Give me a shout if you want to discuss the look and feel. I'll carry on with the rest of the pages.

**Pull request merged**